github3/pulls.py

```diff
@@ -313,14 +313,16 @@ def issue_comments(self, number=-1, etag=None):
         return self._iter(int(number), url, IssueComment, etag=etag)
 
     @requires_auth
-    def merge(self, commit_message='', sha=None):
+    def merge(self, commit_message='', squash=False, sha=None):
```

Comment:
Although isn't this repo preparing for a `1.0` release, would this breaking change not be reasonable?

p.s. just fixed the unit tests, forgot to update unit tests